gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -210,6 +204,13 @@ private void addProjectToNavDrawer(List<Project> projects) {
           .add(R.id.group_join_project, Menu.NONE, index, projects.get(index).getTitle())
           .setIcon(R.drawable.ic_menu_project);
     }
+
+    // Highlight active project
+    Loadable<Project> activeProject = viewModel.getActiveProject().getValue();
+    if (activeProject != null) {
+      int index = getSelectedProjectIndex(activeProject.value().get());
```

Comment:
If the project is not yet activated, `value()` will return null here. Instead, you can replace the  `activeProject` null check with `Loadable.getValue(activeProject)` to cover both cases (i.e. Loadable.getValue(activeProject).ifPresent(p => updateSelectedProjectUI(getSelectedProjectIndex(p)));